lib/deltachat.ts

```diff
@@ -574,9 +574,12 @@ export class DeltaChat extends EventEmitter {
 
   // TODO this should most likely be async, see
   // https://c.delta.chat/classdc__context__t.html#ae49176cbc26d4d40d52de4f5301d1fa7
-  joinSecurejoin (qrCode:string) {
+  joinSecurejoin (qrCode:string, callback: (result: number) => void) {
```

Comment:
I think renaming result to chatId is more helpful even if a chatId of zero means we have an error.